drivers/SmartThings/zwave-switch/src/init.lua

```diff
@@ -79,12 +100,49 @@ local function do_configure(driver, device)
   local configuration = configurationsMap.get_device_configuration(device)
   if configuration ~= nil then
     for _, value in ipairs(configuration) do
-      device:send(Configuration:Set({parameter_number = value.parameter_number, size = value.size, configuration_value = value.configuration_value}))
+      device:send(Configuration:Set({ parameter_number = value.parameter_number, size = value.size, configuration_value = value.configuration_value }))
     end
   end
 end
 
+local function get_profile(command_classes)
+  for _, command_class in pairs(command_classes) do
+    if command_class.value == cc.SENSOR_MULTILEVEL or
+        command_class.value == cc.SENSOR_BINARY or
+        command_class.value == cc.SENSOR_ALARM or
+        command_class.value == cc.NOTIFICATION then
```

Comment:
DTH uses device class here, which should be available in the objects stored in `zwave_endpoints`

Reply:
(on real devices, you'll have to hardcode it for tests)

Reply:
and you'll only want the first two bytes, the generic device class

Reply:
@cjswedes says this might not actually be supported

Reply:
You can get the configuration of each endpoint by sending a multichannel capability get to each endpoint and having a handler for the capability report, since this info isn't accurate on the device object yet